_test/test_mpi/worker_4tests_local.m

```diff
@@ -1,4 +1,4 @@
-function [ok,err_mess,je]=worker_4tests_local(worker_controls_string)
+function [ok,err_mess,je]=worker_v2(worker_controls_string)
```

Comment:
does the file need renaming to match the changed function name?

Reply:
no, it does not. Fixed